types/errors.go

```diff
@@ -30,4 +30,5 @@ var (
 	ErrInvalidFallbackRecipient = errors.Register(ModuleName, 4, "invalid fallback recipient")
 	ErrInvalidDestinationCaller = errors.Register(ModuleName, 5, "invalid destination caller")
 	ErrInvalidTransferAmount    = errors.Register(ModuleName, 6, "invalid autocctp transfer amount")
+	ErrInvalidAmount            = errors.Register(ModuleName, 7, "invalid autocctp account balance")
```

Comment:
Because the codespace is already defined as `autocctp`, doesn't it make sense to remove the `autocctp` from the error message? Also, I would like to align the names better!

```suggestion
	ErrInvalidTransferAmount    = errors.Register(ModuleName, 6, "invalid transfer amount")
	ErrInvalidAccountBalance    = errors.Register(ModuleName, 7, "invalid account balance")
```

Reply:
Agree with both suggestions!